test/index.html

```diff
@@ -38,7 +38,7 @@
          This is the chart description.
     </div>
 
-    <div class="cfpb-chart"
+    <div class="cfpb-chart cfpb-chart__tile-map"
          data-chart-type="tile_map"
```

Comment:
same as above - why not use data attribute to target maps instead of adding a new class?